programs/account-compression/src/instructions/rollover_state_merkle_tree.rs

```diff
@@ -54,37 +54,36 @@ impl<'info> GroupAccounts<'info> for RolloverStateMerkleTreeAndNullifierQueue<'i
 pub fn process_rollover_state_merkle_tree_nullifier_queue_pair<'a, 'b, 'c: 'info, 'info>(
     ctx: Context<'a, 'b, 'c, 'info, RolloverStateMerkleTreeAndNullifierQueue<'info>>,
 ) -> Result<()> {
-    assert_size_equal(
-        &ctx.accounts.old_nullifier_queue.to_account_info(),
-        &ctx.accounts.new_nullifier_queue.to_account_info(),
-    )?;
-    assert_size_equal(
-        &ctx.accounts.old_state_merkle_tree.to_account_info(),
-        &ctx.accounts.new_state_merkle_tree.to_account_info(),
-    )?;
```

Comment:
Please revert this deletion. This check asserts that the forester who performs the rollover doesn't change the account size.